keystrokes/src/stylesheets/character.css

```diff
@@ -0,0 +1,14 @@
+.character-links {
```

Comment:
I couldn't find where these are being referenced, are these old?

Reply:
yea, i replaced them, gotta replace everything else with the new style - ill add that in before the next MR update